doc/api/Loading_a_Content.md

```diff
@@ -706,6 +706,63 @@ Those are the possible values for that option:
   More information about the `"RELOADING"` state can be found in
   [the player states documentation](./Player_States.md).
 
+### onAudioTracksNotPlayable
+
+_type_: `string|undefined`
+
+_defaults_: `"error"`
+
+Specifies the behavior when all audio tracks are not playable - This can occur if the
+device does not support the required audio codecs, or if the content cannot be decrypted,
+for example, due to an insufficient security level.
+
+Those are the possible values for that option:
+
+- `"continue"`: The player will proceed to play the content without audio.
+
+- `"error"`: The player will throw an error `MediaError` with the code
+  `MANIFEST_INCOMPATIBLE_CODECS_ERROR`, `NO_AUDIO_VIDEO_TRACKS` or
+  `NO_PLAYABLE_REPRESENTATION` to indicate that the audio tracks could not be played.
```

Comment:
Maybe we should add details on which error corresponds to what?
In my mind the first is for when no track of a buffer type has supported codec, the second is for when there's no audio nor video track enabled and the third is for when there's no decipherable track for the buffer type. Is that it?